# Pergunta de Pesquisa 3

## Como o consumo de energia elétrica está distribuído entre as regiões de Santa Catarina?

---

## Objetivo

Investigar como o consumo de energia elétrica se distribui entre as diferentes regiões de Santa Catarina, buscando identificar concentrações regionais da demanda energética e possíveis diferenças entre as diversas áreas do estado.

Esta etapa amplia a análise espacial realizada anteriormente, permitindo compreender o consumo sob uma perspectiva regional, em vez da escala municipal.

---

## Motivação

Após compreender como o consumo está distribuído entre os municípios catarinenses e como essa demanda evoluiu ao longo do tempo, o próximo passo é analisar a organização regional do consumo de energia elétrica.

A análise regional permite reduzir a fragmentação observada na escala municipal e facilita a identificação de padrões territoriais associados ao desenvolvimento econômico, à infraestrutura energética e às diferentes características produtivas de Santa Catarina.

Além de aprofundar o entendimento sobre a demanda energética estadual, esta etapa fornecerá uma base importante para investigações futuras envolvendo economia regional, infraestrutura crítica, economia digital e potenciais áreas de interesse para implantação de Data Centers.

---

## Estratégia de Análise

Para responder a esta pergunta serão realizadas análises buscando identificar:

- distribuição regional do consumo de energia elétrica;
- participação relativa de cada região no consumo estadual;
- comparação entre regiões com maior e menor demanda energética;
- padrões espaciais de concentração do consumo;
- primeiras evidências sobre diferenças regionais da infraestrutura energética.

---

## Pergunta de Pesquisa

> **Como o consumo de energia elétrica está distribuído entre as regiões de Santa Catarina?**

---

*Esta análise dá continuidade às Perguntas de Pesquisa 1 e 2. Após investigar a distribuição municipal do consumo de energia elétrica e sua evolução temporal, o foco passa a ser a organização regional da demanda energética, ampliando a compreensão sobre a estrutura territorial do consumo de energia em Santa Catarina.*

In [5]:
import pandas as pd

In [7]:
import os

arquivo = "../data/auxiliary/RELATORIO_DTB_BRASIL_2025_MUNICIPIOS.xls"

print(os.path.exists(arquivo))
print(os.path.getsize(arquivo))

True
1004544


In [9]:
import sys

print(sys.executable)

c:\Users\User\OneDrive\Desktop\Portifolio_Data_Analytics\03_Mercado_Energia_SC\.venv\Scripts\python.exe


In [10]:
import xlrd

print(xlrd.__version__)

2.0.2


In [11]:
ibge = pd.read_excel(
    "../data/auxiliary/RELATORIO_DTB_BRASIL_2025_MUNICIPIOS.xls",
    header=6,
    engine="xlrd",
)

ibge.head()

,UF,Nome_UF,Região Geográfica Intermediária,Nome Região Geográfica Intermediária,Região Geográfica Imediata,Nome Região Geográfica Imediata,Município,Código Município Completo,Nome_Município
0,11,Rondônia,1102,Ji-Paraná,110005,Cacoal,15,1100015,Alta Floresta D'Oeste
1,11,Rondônia,1102,Ji-Paraná,110005,Cacoal,379,1100379,Alto Alegre dos Parecis
2,11,Rondônia,1101,Porto Velho,110002,Ariquemes,403,1100403,Alto Paraíso
3,11,Rondônia,1102,Ji-Paraná,110004,Ji-Paraná,346,1100346,Alvorada D'Oeste
4,11,Rondônia,1101,Porto Velho,110002,Ariquemes,23,1100023,Ariquemes


In [12]:
ibge.info()

<class 'pandas.DataFrame'>
RangeIndex: 5571 entries, 0 to 5570
Data columns (total 9 columns):
 #   Column                                Non-Null Count  Dtype
---  ------                                --------------  -----
 0   UF                                    5571 non-null   int64
 1   Nome_UF                               5571 non-null   str  
 2   Região Geográfica Intermediária       5571 non-null   int64
 3   Nome Região Geográfica Intermediária  5571 non-null   str  
 4   Região Geográfica Imediata            5571 non-null   int64
 5   Nome Região Geográfica Imediata       5571 non-null   str  
 6   Município                             5571 non-null   int64
 7   Código Município Completo             5571 non-null   int64
 8   Nome_Município                        5571 non-null   str  
dtypes: int64(5), str(4)
memory usage: 391.8 KB


In [13]:
ibge.columns.tolist()

['UF',
 'Nome_UF',
 'Região Geográfica Intermediária',
 'Nome Região Geográfica Intermediária',
 'Região Geográfica Imediata',
 'Nome Região Geográfica Imediata',
 'Município',
 'Código Município Completo',
 'Nome_Município']

In [14]:
ibge_sc = ibge[ibge["Nome_UF"] == "Santa Catarina"]

ibge_sc.head()

,UF,Nome_UF,Região Geográfica Intermediária,Nome Região Geográfica Intermediária,Região Geográfica Imediata,Nome Região Geográfica Imediata,Município,Código Município Completo,Nome_Município
4311,42,Santa Catarina,4204,Chapecó,420008,Joaçaba - Herval d'Oeste,51,4200051,Abdon Batista
4312,42,Santa Catarina,4204,Chapecó,420011,Xanxerê,101,4200101,Abelardo Luz
4313,42,Santa Catarina,4207,Blumenau,420022,Rio do Sul,200,4200200,Agrolândia
4314,42,Santa Catarina,4207,Blumenau,420022,Rio do Sul,309,4200309,Agronômica
4315,42,Santa Catarina,4204,Chapecó,420008,Joaçaba - Herval d'Oeste,408,4200408,Água Doce


In [15]:
ibge_sc.shape

(295, 9)

In [16]:
ibge_sc["Nome_Município"].sort_values()

4311       Abdon Batista
4312        Abelardo Luz
4313          Agrolândia
4314          Agronômica
4319      Alfredo Wagner
              ...       
4605              Zortéa
4315           Água Doce
4317         Águas Frias
4318        Águas Mornas
4316    Águas de Chapecó
Name: Nome_Município, Length: 295, dtype: str

In [18]:
ibge

,UF,Nome_UF,Região Geográfica Intermediária,Nome Região Geográfica Intermediária,Região Geográfica Imediata,Nome Região Geográfica Imediata,Município,Código Município Completo,Nome_Município
0,11,Rondônia,1102,Ji-Paraná,110005,Cacoal,15,1100015,Alta Floresta D'Oeste
1,11,Rondônia,1102,Ji-Paraná,110005,Cacoal,379,1100379,Alto Alegre dos Parecis
2,11,Rondônia,1101,Porto Velho,110002,Ariquemes,403,1100403,Alto Paraíso
3,11,Rondônia,1102,Ji-Paraná,110004,Ji-Paraná,346,1100346,Alvorada D'Oeste
4,11,Rondônia,1101,Porto Velho,110002,Ariquemes,23,1100023,Ariquemes
...,...,...,...,...,...,...,...,...,...
5566,52,Goiás,5201,Goiânia,520002,Anápolis,22005,5222005,Vianópolis
5567,52,Goiás,5202,Itumbiara,520009,Piracanjuba,22054,5222054,Vicentinópolis
5568,52,Goiás,5206,Luziânia - Águas Lindas de Goiás,520022,Flores de Goiás,22203,5222203,Vila Boa
5569,52,Goiás,5205,Porangatu - Uruaçu,520018,Ceres - Rialma - Goianésia,22302,5222302,Vila Propício


In [19]:
ibge_sc

,UF,Nome_UF,Região Geográfica Intermediária,Nome Região Geográfica Intermediária,Região Geográfica Imediata,Nome Região Geográfica Imediata,Município,Código Município Completo,Nome_Município
4311,42,Santa Catarina,4204,Chapecó,420008,Joaçaba - Herval d'Oeste,51,4200051,Abdon Batista
4312,42,Santa Catarina,4204,Chapecó,420011,Xanxerê,101,4200101,Abelardo Luz
4313,42,Santa Catarina,4207,Blumenau,420022,Rio do Sul,200,4200200,Agrolândia
4314,42,Santa Catarina,4207,Blumenau,420022,Rio do Sul,309,4200309,Agronômica
4315,42,Santa Catarina,4204,Chapecó,420008,Joaçaba - Herval d'Oeste,408,4200408,Água Doce
...,...,...,...,...,...,...,...,...,...
4601,42,Santa Catarina,4207,Blumenau,420023,Ibirama - Presidente Getúlio,19408,4219408,Witmarsum
4602,42,Santa Catarina,4204,Chapecó,420011,Xanxerê,19507,4219507,Xanxerê
4603,42,Santa Catarina,4204,Chapecó,420011,Xanxerê,19606,4219606,Xavantina
4604,42,Santa Catarina,4204,Chapecó,420007,Chapecó,19705,4219705,Xaxim


In [20]:
arquivo = "../data/raw/Municipio_Mensal_1T_2026.xlsx"

celesc = pd.read_excel(arquivo, sheet_name="Consumo MWh")

In [21]:
celesc.info()

<class 'pandas.DataFrame'>
RangeIndex: 7716 entries, 0 to 7715
Columns: 395 entries, tipo to 2026-03-01 00:00:00
dtypes: float64(386), int64(2), object(1), str(6)
memory usage: 23.3+ MB


In [22]:
celesc.head()

,tipo,cod_muni,agência,núcleo,unidade,município,cod_classe,classe,1994-01-01 00:00:00,1994-02-01 00:00:00,...,2025-06-01 00:00:00,2025-07-01 00:00:00,2025-08-01 00:00:00,2025-09-01 00:00:00,2025-10-01 00:00:00,2025-11-01 00:00:00,2025-12-01 00:00:00,2026-01-01 00:00:00,2026-02-01 00:00:00,2026-03-01 00:00:00
0,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,1,Residencial,21357.147,20782.785,...,52878.556648,60441.735810,54284.838228,58371.440918,53787.994558,53800.328053,56821.529842,87456.494352,86524.998202,83895.217345
1,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,2,Industrial,1698.383,1641.103,...,2558.042630,2877.644839,2557.793542,2761.337071,2972.536314,2669.177959,2677.598859,3332.905398,3222.091413,3746.075399
2,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,3,Comercial,14766.175,13420.879,...,24422.064633,22060.793725,23029.908608,23087.512572,22567.306346,22916.987372,24222.198117,30172.991984,32276.237126,35388.860964
3,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,4,Rural,25.908,23.863,...,37.052000,43.193000,29.520000,37.073000,37.665,38.949800,43.244000,50.350000,33.971450,42.984200
4,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,5,Poder Público,6393.112,4903.650,...,6667.282925,7209.118309,6673.288946,6409.265993,6528.080709,7015.699795,6543.715033,7558.026813,8207.902689,8857.708610


In [23]:
celesc["município"].sort_values().unique()

<StringArray>
[   'Abdon Batista',     'Abelardo Luz',       'Agrolândia',
       'Agronômica',   'Alfredo Wagner',  'Alto Bela Vista',
         'Anchieta',         'Angelina',  'Anita Garibaldi',
   'Antônio Carlos',
 ...
   'Vitor Meireles',        'Witmarsum',          'Xanxerê',
        'Xavantina',            'Xaxim',           'Zortéa',
        'Água Doce',      'Águas Frias',     'Águas Mornas',
 'Águas de Chapecó']
Length: 290, dtype: str

In [24]:
len(celesc["município"].unique())

290

In [25]:
municipios_celesc = set(celesc["município"].str.upper())
municipios_ibge = set(ibge_sc["Nome_Município"].str.upper())

In [26]:
municipios_ibge - municipios_celesc

{'ANITÁPOLIS',
 'BALNEÁRIO PIÇARRAS',
 'BALNEÁRIO RINCÃO',
 'CAIBI',
 'CAXAMBU DO SUL',
 'ENTRE RIOS',
 'FLOR DO SERTÃO',
 'GRÃO-PARÁ',
 'GUATAMBÚ',
 'LAJEADO GRANDE',
 'LAURO MÜLLER',
 'MORRO GRANDE',
 'PRAIA GRANDE',
 'PRESIDENTE CASTELLO BRANCO',
 'RIO FORTUNA',
 'SANTA ROSA DE LIMA',
 'SÃO BONIFÁCIO',
 'SÃO LOURENÇO DO OESTE',
 'SÃO LUDGERO',
 'SÃO MIGUEL DO OESTE',
 'TIMBÉ DO SUL'}

In [27]:
municipios_celesc - municipios_ibge

{'BARRACÃO',
 'CAIBÍ',
 'CAXAMBÚ DO SUL',
 'FLOR SERRA DO SUL',
 'FLÔR DO SERTÃO',
 'GUATAMBU',
 'LAGEADO GRANDE',
 'LAURO MULLER (ARCRI)',
 'LAURO MULLER (ARTUB)',
 'MARMELEIRO',
 'PIÇARRAS',
 'PRESIDENTE CASTELO BRANCO',
 'RENASCENÇA',
 'RIO NEGRO',
 "SÃO LOURENÇO D'OESTE",
 "SÃO MIGUEL D'OESTE"}

In [28]:
len(municipios_ibge - municipios_celesc)

21

In [29]:
len(municipios_celesc - municipios_ibge)

16

In [30]:
sorted(municipios_celesc - municipios_ibge)

['BARRACÃO',
 'CAIBÍ',
 'CAXAMBÚ DO SUL',
 'FLOR SERRA DO SUL',
 'FLÔR DO SERTÃO',
 'GUATAMBU',
 'LAGEADO GRANDE',
 'LAURO MULLER (ARCRI)',
 'LAURO MULLER (ARTUB)',
 'MARMELEIRO',
 'PIÇARRAS',
 'PRESIDENTE CASTELO BRANCO',
 'RENASCENÇA',
 'RIO NEGRO',
 "SÃO LOURENÇO D'OESTE",
 "SÃO MIGUEL D'OESTE"]

In [31]:
sorted(municipios_ibge - municipios_celesc)

['ANITÁPOLIS',
 'BALNEÁRIO PIÇARRAS',
 'BALNEÁRIO RINCÃO',
 'CAIBI',
 'CAXAMBU DO SUL',
 'ENTRE RIOS',
 'FLOR DO SERTÃO',
 'GRÃO-PARÁ',
 'GUATAMBÚ',
 'LAJEADO GRANDE',
 'LAURO MÜLLER',
 'MORRO GRANDE',
 'PRAIA GRANDE',
 'PRESIDENTE CASTELLO BRANCO',
 'RIO FORTUNA',
 'SANTA ROSA DE LIMA',
 'SÃO BONIFÁCIO',
 'SÃO LOURENÇO DO OESTE',
 'SÃO LUDGERO',
 'SÃO MIGUEL DO OESTE',
 'TIMBÉ DO SUL']

In [32]:
correcao = {
    "PICARRAS": "BALNEÁRIO PIÇARRAS",
    "SÃO LOURENÇO D'OESTE": "SÃO LOURENÇO DO OESTE",
    "SÃO MIGUEL D'OESTE": "SÃO MIGUEL DO OESTE",
    "PRESIDENTE CASTELO BRANCO": "PRESIDENTE CASTELLO BRANCO",
    "RIO NEGRO": "RIO NEGRINHO",
}

In [33]:
celesc[celesc["município"].str.contains("RIO", case=False)]

,tipo,cod_muni,agência,núcleo,unidade,município,cod_classe,classe,1994-01-01 00:00:00,1994-02-01 00:00:00,...,2025-06-01 00:00:00,2025-07-01 00:00:00,2025-08-01 00:00:00,2025-09-01 00:00:00,2025-10-01 00:00:00,2025-11-01 00:00:00,2025-12-01 00:00:00,2026-01-01 00:00:00,2026-02-01 00:00:00,2026-03-01 00:00:00
207,Cativo,2107,ARBLU,Núcleo Alto Vale,Blumenau (sede),Rio dos Cedros,1,Residencial,229.658,265.082,...,887.087867,1015.041480,859.156767,899.106796,930.158535,885.769616,1019.843772,1302.630171,1219.551026,1197.662670
208,Cativo,2107,ARBLU,Núcleo Alto Vale,Blumenau (sede),Rio dos Cedros,2,Industrial,111.014,147.737,...,392.984709,399.221776,400.941333,390.518138,318.286047,322.262822,285.984012,263.026196,338.312790,388.011003
209,Cativo,2107,ARBLU,Núcleo Alto Vale,Blumenau (sede),Rio dos Cedros,3,Comercial,50.239,60.299,...,386.554070,345.170853,372.378193,312.570881,315.740342,372.862627,324.507364,380.004954,385.780084,391.889901
210,Cativo,2107,ARBLU,Núcleo Alto Vale,Blumenau (sede),Rio dos Cedros,4,Rural,172.309,202.675,...,242.226996,255.329800,254.864155,274.763500,256.479173,264.526877,314.469050,307.400460,295.433400,274.716060
211,Cativo,2107,ARBLU,Núcleo Alto Vale,Blumenau (sede),Rio dos Cedros,5,Poder Público,8.099,7.902,...,41.133068,38.800106,49.348050,42.594413,40.360638,43.565496,51.208478,50.366121,58.198898,93.302650
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
7678,Livre,17106,ARITA,Núcleo Leste,Itajaí (sede),Balneário Camboriú,3,Comercial,NaN,NaN,...,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,75.321648
7679,Livre,17106,ARITA,Núcleo Leste,Itajaí (sede),Balneário Camboriú,3,Comercial,NaN,NaN,...,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,71.596800
7680,Livre,15102,ARMAF,Núcleo Norte,Mafra (unidade),Rio Negro,2,Industrial,NaN,NaN,...,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,78.548064
7690,Livre,17106,ARITA,Núcleo Leste,Itajaí (sede),Balneário Camboriú,3,Comercial,NaN,NaN,...,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,32.379528


In [34]:
celesc[celesc["município"] == "Rio Negro"][["cod_muni", "município"]]

,cod_muni,município
2043,15102,Rio Negro
2044,15102,Rio Negro
2045,15102,Rio Negro
2046,15102,Rio Negro
2047,15102,Rio Negro
2048,15102,Rio Negro
2049,15102,Rio Negro
2050,15102,Rio Negro
2051,15102,Rio Negro
2950,15102,Rio Negro


In [35]:
celesc[celesc["município"].str.contains("NEGR", case=False)][
    ["agência", "núcleo", "unidade", "município"]
].drop_duplicates()

,agência,núcleo,unidade,município
495,ARLAG,Núcleo Planalto,Lages (sede),Cerro Negro
2043,ARMAF,Núcleo Norte,Mafra (unidade),Rio Negro
2160,ARSBS,Núcleo Norte,São Bento do Sul (unidade),Rio Negrinho


In [36]:
celesc[celesc["município"] == "Rio Negro"]["tipo"].unique()

<StringArray>
['Cativo', 'Livre']
Length: 2, dtype: str

In [37]:
celesc[celesc["município"] == "Rio Negro"][
    ["agência", "núcleo", "unidade", "classe"]
].drop_duplicates()

,agência,núcleo,unidade,classe
2043,ARMAF,Núcleo Norte,Mafra (unidade),Residencial
2044,ARMAF,Núcleo Norte,Mafra (unidade),Industrial
2045,ARMAF,Núcleo Norte,Mafra (unidade),Comercial
2046,ARMAF,Núcleo Norte,Mafra (unidade),Rural
2047,ARMAF,Núcleo Norte,Mafra (unidade),Poder Público
2048,ARMAF,Núcleo Norte,Mafra (unidade),Iluminação Pública
2049,ARMAF,Núcleo Norte,Mafra (unidade),Serviço Público
2050,ARMAF,Núcleo Norte,Mafra (unidade),Próprio
2051,ARMAF,Núcleo Norte,Mafra (unidade),Revenda


### Tratamento de inconsistências entre bases

Durante a comparação entre a base de consumo da CELESC e a Divisão Territorial Brasileira (IBGE), foi identificado que a CELESC possui registros para o município de Rio Negro (PR), município que não integra o estado de Santa Catarina.

Como esta pesquisa está restrita aos municípios catarinenses, esses registros serão removidos antes da integração entre as bases, garantindo consistência espacial nas análises seguintes.

In [38]:
celesc = celesc[celesc["município"] != "Rio Negro"].copy()

In [40]:
ibge_sc.columns.tolist()

['UF',
 'Nome_UF',
 'Região Geográfica Intermediária',
 'Nome Região Geográfica Intermediária',
 'Região Geográfica Imediata',
 'Nome Região Geográfica Imediata',
 'Município',
 'Código Município Completo',
 'Nome_Município']

In [42]:
celesc.columns.tolist()

['tipo',
 'cod_muni',
 'agência',
 'núcleo',
 'unidade',
 'município',
 'cod_classe',
 'classe',
 datetime.datetime(1994, 1, 1, 0, 0),
 datetime.datetime(1994, 2, 1, 0, 0),
 datetime.datetime(1994, 3, 1, 0, 0),
 datetime.datetime(1994, 4, 1, 0, 0),
 datetime.datetime(1994, 5, 1, 0, 0),
 datetime.datetime(1994, 6, 1, 0, 0),
 datetime.datetime(1994, 7, 1, 0, 0),
 datetime.datetime(1994, 8, 1, 0, 0),
 datetime.datetime(1994, 9, 1, 0, 0),
 datetime.datetime(1994, 10, 1, 0, 0),
 datetime.datetime(1994, 11, 1, 0, 0),
 datetime.datetime(1994, 12, 1, 0, 0),
 datetime.datetime(1995, 1, 1, 0, 0),
 datetime.datetime(1995, 2, 1, 0, 0),
 datetime.datetime(1995, 3, 1, 0, 0),
 datetime.datetime(1995, 4, 1, 0, 0),
 datetime.datetime(1995, 5, 1, 0, 0),
 datetime.datetime(1995, 6, 1, 0, 0),
 datetime.datetime(1995, 7, 1, 0, 0),
 datetime.datetime(1995, 8, 1, 0, 0),
 datetime.datetime(1995, 9, 1, 0, 0),
 datetime.datetime(1995, 10, 1, 0, 0),
 datetime.datetime(1995, 11, 1, 0, 0),
 datetime.datetime(1995,

In [43]:
municipios_ibge = set(ibge_sc["Nome_Município"])
municipios_celesc = set(celesc["município"])

print(len(municipios_ibge))
print(len(municipios_celesc))
print(municipios_ibge == municipios_celesc)

295
289
False


In [44]:
sorted(municipios_ibge - municipios_celesc)

['Anitápolis',
 'Balneário Piçarras',
 'Balneário Rincão',
 'Caibi',
 'Caxambu do Sul',
 'Entre Rios',
 'Flor do Sertão',
 'Grão-Pará',
 'Guatambú',
 'Lajeado Grande',
 'Lauro Müller',
 'Morro Grande',
 'Praia Grande',
 'Presidente Castello Branco',
 'Rio Fortuna',
 'Santa Rosa de Lima',
 'São Bonifácio',
 'São Lourenço do Oeste',
 'São Ludgero',
 'São Miguel do Oeste',
 'Timbé do Sul']

### Cobertura da base de consumo

Após o tratamento das inconsistências, verificou-se que a base de consumo da CELESC contempla 289 municípios, enquanto a Divisão Territorial Brasileira (IBGE) possui 295 municípios em Santa Catarina.

As análises seguintes serão realizadas considerando apenas os municípios presentes na base de consumo.

In [45]:
celesc.shape

(7681, 395)

In [46]:
base_sc = celesc.merge(
    ibge_sc, left_on="cod_muni", right_on="Código Município Completo", how="left"
)

In [47]:
base_sc.shape

(7681, 404)

In [48]:
base_sc.head()

,tipo,cod_muni,agência,núcleo,unidade,município,cod_classe,classe,1994-01-01 00:00:00,1994-02-01 00:00:00,...,2026-03-01 00:00:00,UF,Nome_UF,Região Geográfica Intermediária,Nome Região Geográfica Intermediária,Região Geográfica Imediata,Nome Região Geográfica Imediata,Município,Código Município Completo,Nome_Município
0,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,1,Residencial,21357.147,20782.785,...,83895.217345,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN
1,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,2,Industrial,1698.383,1641.103,...,3746.075399,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN
2,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,3,Comercial,14766.175,13420.879,...,35388.860964,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN
3,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,4,Rural,25.908,23.863,...,42.984200,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN
4,Cativo,1101,ARFLO,Núcleo Grande Capital,Florianópolis (sede),Florianópolis,5,Poder Público,6393.112,4903.650,...,8857.708610,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN
